core/src/components/toast/test/a11y/toast.e2e.ts

```diff
@@ -38,5 +38,26 @@ configs({ directions: ['ltr'] }).forEach(({ title, config }) => {
       const results = await new AxeBuilder({ page }).disableRules('color-contrast').analyze();
       expect(results.violations).toEqual([]);
     });
+
+    test('should have aria-labelledby and aria-label added to the button when htmlAttributes is set', async ({
+      page,
+    }) => {
+      const didPresent = await page.spyOnEvent('ionToastDidPresent');
+
+      await page.click('#aria-label-toast-trigger');
+      await didPresent.next();
+
+      const toastButton = page.locator('#aria-label-toast .toast-button');
+
+      /**
+       * expect().toHaveAttribute() can't check for a null value, so grab and check
```

Comment:
Same as https://github.com/ionic-team/ionic-framework/pull/27862/files#r1276705971:

Why can't we use the `toHaveAttribute` API? We aren't checking for null values, so that limitation should not apply here.

Reply:
Fixed by https://github.com/ionic-team/ionic-framework/pull/27855/commits/353d2b9bf2f847b3be5ac078b3eb05cea52c2117